model/MMA8453.cpp

```diff
@@ -0,0 +1,255 @@
+#include "MMA8453.h"
+#include "CodalConfig.h"
+#include "ErrorNo.h"
+#include "CodalCompat.h"
+#include "CodalFiber.h"
+
+using namespace codal;
+
+//
+// Configuration table for available g force ranges.
+// Maps g -> LIS3DH_CTRL_REG4 [5..4]
+//
+static const KeyValueTableEntry accelerometerRangeData[] = {
+    {2, 0},
+    {4, 1},
+    {8, 2},
+ 
+};
+CREATE_KEY_VALUE_TABLE(accelerometerRange, accelerometerRangeData);
+
+
+/**
```

Comment:
Were all these doc comments taken from some base class? are they still correct? I see references to `LIS3DH` for example